README.md

```diff
@@ -3,6 +3,12 @@ Pi Video Machine - a scalable, synchronized, and networked-controlled, raspberry
 
 
 ### { This is a work in progress }
+  
+- TODO: rename the variable
```

Comment:
@KaneBetter what variable?

Reply:
I mean `media` in https://github.com/omarcostahamido/PVM/blob/UseOmxplayer/pvm.py#L26

Reply:
Ok, I see what you did now. You just transcribed all the todo comment lines into the readme.